TIGLViewer/src/TIGLViewerApp.cpp

```diff
@@ -94,6 +95,9 @@ int TIGLViewerApp::parseArguments(QStringList argList)
                 config.initialScript = argList.at(++i);
             }
         }
+        else if (arg.compare("--suppress-errors") == 0) {
+            config.suppress_errors = true;
```

Comment:
`suppress_errors` is not TiGL Style conform. Please chaneg it to `suppressErrors`